src/main/java/fm/last/moji/tracker/pool/ManagedTrackerHost.java

```diff
@@ -58,7 +58,7 @@
   }
 
   ManagedTrackerHost(InetSocketAddress address) {
-    this(address, new Timer(), Clock.INSTANCE);
+    this(address, new Timer(true), Clock.INSTANCE);
```

Comment:
Looks OK to me, @hrchu any thoughts on this based on how you're using moji? Are there any use cases where one would want the timer to prolong the life of the application? If someone did want that I think they should probably do it within their app rather than having it be a side effect of this Timer.